packages/graph/hash_graph/lib/graph/Cargo.toml

```diff
@@ -37,6 +37,7 @@ type-system = { git = "https://github.com/blockprotocol/blockprotocol", rev = "2
 uuid = { version = "1.2.2", features = ["v4", "serde"] }
 utoipa = { version = "2.4.2", features = ["uuid"] }
 include_dir = "0.7.3"
+refinery = { version = "0.8", features = ["postgres", "tokio-postgres"]}
```

Comment:
Do we actually need both, `"postgres"` _and_ `"tokio-postgres"`?

Reply:
Yep, from their readme:

"You will still need to provide the postgres/mysql/rusqlite/tiberius driver as a feature for [Runner::run](https://docs.rs/refinery/latest/refinery/struct.Runner.html#method.run) and tokio-postgres/mysql_async"

Reply:
8b7d234495f663221b99d3c8863c57ff11edc008 Thanks!